opentelemetry-sdk/src/metrics/periodic_reader.rs

```diff
@@ -131,6 +131,12 @@ where
             }));
         };
 
+        otel_debug!(
+            name: "PeriodicReaderBuilt",
```

Comment:
We can be consistent with the other event names and follow this format: `{componentName}.{action specified in past tense form}`

```suggestion
            name: "PeriodicReader.BuildCompleted",
```